e2e/helpers/src/bin/start_committer.rs

```diff
@@ -0,0 +1,82 @@
+use anyhow::Result;
+use std::time::Duration;
+use tokio::time::sleep;
+use url::Url;
+
+use e2e_helpers::{
+    committer::Committer,
+    eth_node::{EthNode, EthNodeProcess},
+    kms::{Kms, KmsProcess},
+    whole_stack::{create_and_fund_kms_signers, deploy_contract},
+};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let kms = start_kms(false).await?;
+
+    let eth_node = start_eth(false).await?;
+    let eth_signers = create_and_fund_kms_signers(&kms, &eth_node).await?;
+    let eigen_key = env!("EIGEN_KEY").to_string();
```

Comment:
nit: same here; unsure whether you want this to be a compile-time check instead of runtime